app/lib/adapter/bimaskemenag.ts

```
@@ -1,17 +1,25 @@
 import got from "got";
 import { JSDOM } from 'jsdom';
 
+var cookieCache: { cookies: string, timestamp: number } | null = null;
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)